src/deployments/kubernetes/README.md

```diff
@@ -0,0 +1,66 @@
+# Kubernetes Deployment
+
+This folder provides a basic setup of a set of
+[Kubernetes](https://kubernetes.io/) files to setup a cluster.
+
+## Local Development
+
+For local development, you need to use
+[minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+* After installing, run `minikube start` to setup minikube in your local
+  environment.
+* Set the docker environment inside minikube:
+
+`eval $(minikube docker-env)` for Unix shells or
+`minikube docker-env | Invoke-Expression` in PowerShell
+
+* From the location of the [Dockerfile](src\Dockerfile), Build the data management service image:
+`docker build -t local/edfi-data-management-service:latest .`
+
+> [!WARNING]
+> The [deployment file](./data-management-service-deployment.yaml#L21) has a policy
+> of Never pull, which means that will use the locally built image instead of trying
+> to pull the image from Docker Hub which is the default behavior, remove
+> once the image is published to the registry.
+
+* Set the terminal in the */deployments/kubernetes* folder.
+* Run `kubectl apply -f .` to apply all files.
```

Comment:
![image](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/assets/9324390/bd7f6d6a-3c99-4535-afc5-deebc234ce36)

This is what I'm getting right now. Might be a problem specific to my computer or to the MSDF office network.

Reply:
I am able to follow and set up the application. 
![Screenshot 2024-03-07 100552](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/assets/34072978/b12fc4b4-d134-494f-a4d0-78ff19202430)